doc/SUMMARY.md

```diff
@@ -2,31 +2,31 @@
 
 [Introduction](./introduction.md)
 
+- [General Concepts](./concepts/introduction.md)
+    - [Blockchain concepts](./concepts/blockchain.md)
+    - [Node organisation](./concepts/node.md)
+    
 - [Quickstart](./quickstart/introduction.md)
     - [Command lines tools](./quickstart/01_command_line.md)
     - [Starting a passive node](./quickstart/02_passive_node.md)
     - [REST API](./quickstart/03_rest_api.md)
 
-- [General Concepts](./concepts/introduction.md)
-    - [Blockchain concepts](./concepts/blockchain.md)
-    - [Node organisation](./concepts/node.md)
-
 - [Configuration](./configuration/introduction.md)
     - [Logging](./configuration/logging.md)
     - [Node network](./configuration/network.md)
 
-- [staking and stake pool](./stake_pool/introduction.md)
-    - [registering your stake](./stake_pool/registering_stake.md)
-    - [delegating your stake](./stake_pool/delegating_stake.md)
-    - [registering stake pool](./stake_pool/registering_stake_pool.md)
-
 - [jcli](./jcli/introduction.md)
     - [Cryptographic keys](./jcli/key.md)
     - [Address](./jcli/address.md)
     - [Transaction](./jcli/transaction.md)
     - [Certificate](./jcli/certificate.md)
     - [Genesis](./jcli/genesis.md)
     - [REST](./jcli/rest.md)
+    
+    - [staking and stake pool](./stake_pool/introduction.md)
```

Comment:
this cannot be part of the JCLI chapter.

have you tried generating the output of this? It is pretty straightforward: [install mdbook](https://github.com/rust-lang-nursery/mdBook#installation)

Reply:
can we move the staking chapter below the JCLI chapter please

Reply:
I think the indentation spaces here are just a typo. 

@nahern , could you remove the indentation spaces from this line?

Apart from that, I think this PR is ready to be accepted. But the Jormungandr's team should have the final decision.

Reply:
this PR is to re-order sections in this doc, to move the JCLI chapter ahead of the stake pool chapter, indentation is a mistake, and I can't seem to edit this.